src/utils.jl

```diff
@@ -33,8 +35,9 @@ end
 
 function _filter_indexed(missed_, vars)
     missed = Number[]
+    vars_and_already_missed = vars
     for m in missed_
-        _in_without_index(m, vars) || push!(missed, m)
+        _in_without_index(m, vars_and_already_missed) || (push!(missed, m); push!(vars_and_already_missed, [m, adjoint(m)]...))
     end
```

Comment:
I'd skip defining `vars_and_already_missed` here.
Just change the check to `_in_without_index(m, vars) || _in_without_index(m, missed) || push!(missed, m)`.